main.tf

```
@@ -28,6 +28,28 @@ resource "aws_vpc" "myvpc" {
   }
 }
 
+resource "aws_default_security_group" "default" {
+  vpc_id = aws_vpc.myvpc.id
+
+  ingress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "${var.env_code}SecurityGrp"
```

Later you will have more than one security group
It will make sense to differentiate it
```suggestion
    Name = "${var.env_code}DefaultSecurityGrp"
```